### ISSUE TITLE
it workers Mac book air 2017?

### ISSUE THREAD

**Issue by diegolaz73** [pookjw/SidecarPatcher#95]

**Comment by diegolaz73** [comment]
idk if it is the same patch...

**Comment by hasantha-nirmal** [comment]
Yes, it does. I used this in my only mac, macbook air 2017. Unfortunately no matter connection is wireless or wired, it's so glitchy, the visual. So ended up removing it.
